src/diff/index.js

```diff
@@ -25,8 +25,8 @@ import options from '../options';
  * render (except when hydrating). Can be a sibling DOM element when diffing
  * Fragments that have siblings. In most cases, it starts out as `oldChildren[0]._dom`.
  */
-export function diff(dom, parentDom, newVNode, oldVNode, context, isSvg, excessDomChildren, mounts, ancestorComponent, force, oldDom) {
-
+export function diff(dom, parentDom, newVNode, oldVNode, context, isSvg, excessDomChildren, mounts, ancestorComponent, force, oldDom, depth) {
+	if (!depth) depth = 0;
```

Comment:
Hmm would the size be any smaller if instead of adding an argument to `diff` we just set it directly on each vnode as we loop through them in `diffChildren`?

Something like: (pseudo-code of `diffChildren`)
```js
function diffChildren(newParentVNode) {
    for (let child of newParentVNode._children) {
        // The initial VNode is always a Fragment that won't have `_depth` set
        child._depth = (newParentVNode._depth | 0) + 1;
    }
}
```
It's one benefit of `diffChildren` being the entry point into the diff algorithm 😁

Reply:
Problem here being that you won't be able to set your root, or we should set it there aswell.

Reply:
Just tried that and it saves `5 B`. All tests are passing too.

Reply:
Yes but demo will still fail, can’t repro a test for this atm

Reply:
True, I stand corrected! Thanks for clearing this up :+1: